Alloy/lib/alloy/controllers/BaseController.js

```diff
@@ -219,9 +350,36 @@ var Controller = function() {
 
 		/**
 		 * @method updateViews
-		 * Applies a set of properties to view elements associated with this controller
+		 * Applies a set of properties to view elements associated with this controller.
 		 * @param {Object} args An object whose keys are the IDs (in form '#id') of views to which the styles will be applied.
 		 * @since 1.4.0
+		 * #### Example
```

Comment:
This should probably go before @param.  It's being appended to the `args` parameter description.